app/controllers/api/households_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Api
+  # Exposes the Household data
+  class HouseholdsController < Api::BaseController
+    before_action :set_household, only: %i[show update delete]
+    # GET /households
+    # def index
+    #   # Should get household by current user_id/member_id
+    #   @household = Household.joins(:members)
+    #   .where(members: { id: current_user.member.id })
+    #   render json: @household
+    # end
+
+    # GET /households/1
+    def show
+      if @household
+        render json: @household
+      else
+        render json: {}, status: :not_found
+      end
+    end
+
+    # POST /households
+    def create
+      @household = Household.new(household_params)
+      set_added_by
+      if @household.save
+        render json: @household
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    # PUT /households/1
+    def update
+      if @household.update(household_params)
+        render json: @household
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    # DELETE /households/1
+    def delete
+      if @household.destroy
+        render json: { deleted: true }
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    def set_added_by
+      @household.added_by = current_user.id
+      @household.last_updated_by = current_user.id
+      return unless @household.address
+
+      @household.address.added_by = current_user.id
+      @household.address.last_updated_by = current_user.id
+    end
+
+    def set_household
+      @household = Household.find_by(id: params[:id])
```

Comment:
Once we have the concept of a household belonging to a user (see comment on the migration), the search should include the `current_user.id` to prevent folks from being able to view or edit households that don't belong to them.